click_odoo_contrib/initdb.py

```diff
@@ -12,13 +12,14 @@
 
 import click
 import click_odoo
-from click_odoo import odoo
+from click_odoo import odoo, parse_version
 
 from ._dbutils import db_exists
 from .manifest import expand_dependencies
 from .update import _save_installed_checksums
 
 _logger = logging.getLogger(__name__)
+_odoo_version = parse_version(odoo.release.version)
```

Comment:
Another argument for not exposing it in click_odoo is that this lines mixes click_odoo's parse_version with something from the internal Odoo API (odoo.release.version). So there is not real benefit in having parse_version alone exposed in click_odoo.